### PR TITLE
CHI-1287: Upgrade CLI & deps

### PR DESCRIPTION
**Pull request by stephenhand**
Primary reviewer: @GPaoloni 

## Description

* Updated Flex 2.0.0-beta.3
* Updated to TS 4
* Update CLI to 5.2.2 / 6.0.3

### Checklist
- [X] Corresponding issue has been opened
- [X] New tests added
- N/A Feature flags added
- N/A Strings are localized
- [X] Tested for chat contacts
- [X] Tested for call contacts

### Related Issues
Fixes CHI-1287

### Verification steps

General high level regression
Run unit tests

### CODE REVIEW

File: plugin-hrm-form/src/components/twilioComponentWorkaround.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Button as TwilioButton, IconButton as TwilioIconButton } from '@twilio/flex-ui';
```
Comment (stephenhand) [inline plugin-hrm-form/src/components/twilioComponentWorkaround.ts:1]: Ticket open for this: https://console.twilio.com/us1/support/tickets?frameUrl=%2Fconsole%2Fsupport%2Ftickets%2Fcreate%3F__override_layout__%3Dembed%26bifrost%3Dtrue%26x-target-region%3Dus1&currentFrameUrl=%2Fconsole%2Fsupport%2Ftickets%2F10121147%3F__override_layout__%3Dembed%26bifrost%3Dtrue%26x-target-region%3Dus1

**Review by GPaoloni — APPROVED** [review]
Looks and works great!

File: plugin-hrm-form/src/components/twilioComponentWorkaround.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * This is a workaround for an issue seen starting in @twilio/flex-ui v2.0.0-beta.3
+ * Twilio Buttons were failing on type checks because it thought their props has a mandatory 'css' property, which it doesn't
+ * This workaround does some TS back flips to create the same type with the 'css' property explicitly omitted, which shuts the TS compiler up
+ * Just use this instead of the Twilio UI button as long as the issue persists
+ */
+
+type extractGeneric<Type> = Type extends React.FC<infer X> ? X : never;
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/twilioComponentWorkaround.ts:10]: Shutout to best TS line of the repo :rocket:

Comment (stephenhand) [reply plugin-hrm-form/src/components/twilioComponentWorkaround.ts:10]: Shoutout to StackOverflow :-P Pretty cool that you can pull the generic parameter out of a type like that though

**Pull request merged**